### PR TITLE
feat : cart component

### PR DESCRIPTION
**Pull request by arpit01923**
-Cart Page

1. Cart page tells about products added to cart and having three components Navigation, Main and Footer.
2. Navigation component is used to navigate logo, search, profile, wishlist and cart page, and for login or signup.
3. Main component is used for displaying cart and products present in cart.
4. Footer component is used for giving some extra information.

-What files to check :

1. src/pages/cart/Cart.jsx
2.src/pages/cart/main/Main.jsx

-Preview

https://user-images.githubusercontent.com/86866251/159480357-dba6c92b-6db3-4dca-be24-f0b818ac3778.mp4

### CODE REVIEW

File: src/pages/cart/Cart.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import { Footer, Navbar } from "../../components";
+import "./Cart.css"
+import { Main } from "./main/Main";
```
Comment (anupkgurung) [inline src/pages/cart/Cart.jsx:2]: try using relative path instead of absolute path

**Review by anupkgurung — COMMENTED** [review]
try to use relative path.
Also main jsx will not have price and other properties ? as i can only see data is providing only images.

File: src/App.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 import "./App.css";
-import { Product } from "./pages";
+import { Cart } from "./pages";
 
 function App() {
   return (
```
Comment (rohanmathur91) [inline src/App.js:5]: try to follow any one convention for functions

**Comment by rohanmathur91** [comment]
- Try to give some spacing between the wishlist icon and the text in add to wishlist button of your card
- Code structure is good
- For folder structuring, you are having Main component inside your home page folder, so put Main in a component folder,
pages folder should only contain page.

File: src/pages/cart/Cart.jsx (hunk 1 above)
Comment (Arpit-dotcom) [reply src/pages/cart/Cart.jsx:3]: used absolute path instead of relative path

**Pull request merged**